### PR TITLE
docs: improve the installation guide in getting start page

### LINKED ISSUE #204: feat(docs): Add more details about the installation
> @kemingy Yeah, we can add an installation section to [Building Your First Environment page](https://envd.tensorchord.ai/guide/build-envd.html)(P.S. the page title is different from sidebar entry :P), or in its own page. We can then make it more verbose and clear there, and leave a link referring to the docs in README. What's more, we can make a more beautiful UI for it, such as a tab component to switch different installation methods.

_Originally posted by @frostming in https://github.com/tensorchord/envd/issues/1300#issuecomment-1348192479_

### PR DESCRIPTION
**Pull request by frostming**
- Add a (simple) installation script to the root of the website so that one can install envd with one command, without any tool except for `curl`
- Update the docs and chinese docs to list several installation methods

Resolve #204

### CODE REVIEW

File: docs-zh/guide/getting-started.md [hunk 1]
```diff
@@ -110,18 +110,55 @@ def build():
...
+在终端中运行以下命令即可安装最新版本的 `envd`:
+
+```bash
+curl -sSfL https://envd.tensorchord.ai/install.sh | sudo sh
+```
+
+</tab>
+<tab name="Homebrew">
+
+如果你使用的是 MacOS，可以通过 [homebrew](https://brew.sh/) 来安装 `envd`。
```
Comment (gaocegege) [inline docs-zh/guide/getting-started.md:133]: Should we add a github action as https://github.com/Homebrew/homebrew-core/pull/118130#discussion_r1048501538 said?

**Comment by gaocegege** [comment]
```
6:13:05 PM: build error:
6:13:05 PM:  file:///opt/build/repo/docs/.vitepress/.temp/app.js:4
6:13:05 PM: import { Tabs, Tab } from "vue3-tabs-component";
6:13:05 PM:                ^^^
6:13:05 PM: SyntaxError: Named export 'Tab' not found. The requested module 'vue3-tabs-component' is a CommonJS module, which may not support all module.exports as named exports.
6:13:05 PM: CommonJS modules can always be imported via the default export, for example using:
6:13:05 PM: import pkg from 'vue3-tabs-component';
6:13:05 PM: const { Tabs, Tab } = pkg;
6:13:05 PM:     at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
6:13:05 PM:     at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)
6:13:05 PM:     at async Promise.all (index 0)
6:13:05 PM:     at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
6:13:05 PM:     at async build (file:///opt/build/repo/node_modules/.pnpm/vitepress@1.0.0-alpha.9_ygzyjl425qrcgm3zeayiliwge4/node_modules/vitepress/dist/node/serve-dfb48091.js:40929:24)
```

You could get the log with `pnpm build`

File: docs-zh/guide/getting-started.md (hunk 1 above)
Comment (frostming) [reply docs-zh/guide/getting-started.md:133]: No, I think most package maintainers don't do that. The homebrew maintainers are very active and there is a bot @BrewTestBot doing that.

**Comment by frostming** [comment]
> ```
> 6:13:05 PM: build error:
> 6:13:05 PM:  file:///opt/build/repo/docs/.vitepress/.temp/app.js:4
> 6:13:05 PM: import { Tabs, Tab } from "vue3-tabs-component";
> 6:13:05 PM:                ^^^
> 6:13:05 PM: SyntaxError: Named export 'Tab' not found. The requested module 'vue3-tabs-component' is a CommonJS module, which may not support all module.exports as named exports.
> 6:13:05 PM: CommonJS modules can always be imported via the default export, for example using:
> 6:13:05 PM: import pkg from 'vue3-tabs-component';
> 6:13:05 PM: const { Tabs, Tab } = pkg;
> 6:13:05 PM:     at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
> 6:13:05 PM:     at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)
> 6:13:05 PM:     at async Promise.all (index 0)
> 6:13:05 PM:     at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
> 6:13:05 PM:     at async build (file:///opt/build/repo/node_modules/.pnpm/vitepress@1.0.0-alpha.9_ygzyjl425qrcgm3zeayiliwge4/node_modules/vitepress/dist/node/serve-dfb48091.js:40929:24)
> ```
> 
> You could get the log with `pnpm build`

Got it, I could seek for another tab component library :(

File: docs-zh/guide/getting-started.md (hunk 1 above)
Comment (gaocegege) [reply docs-zh/guide/getting-started.md:133]: Then, for example, we already had a new release v0.2.6. How do we update it in homebrew?

**Comment by gaocegege** [comment]
/cc @VoVAllen @kemingy 

They know more about frontend.

File: docs-zh/guide/getting-started.md (hunk 1 above)
Comment (frostming) [reply docs-zh/guide/getting-started.md:133]: We can wait for half a day to see if someone has submitted an update PR 😆 .  And we'll see whether we need to setup the update workflow ourselves.

Comment (gaocegege) [reply docs-zh/guide/getting-started.md:133]: SGTM!

**Comment by frostming** [comment]
The build error seems to be cleared, except for the existing dead links. As the PR preview is unavailable, here is the screenshot:

image

File: docs-zh/guide/getting-started.md (hunk 1 above)
Comment (frostming) [reply docs-zh/guide/getting-started.md:133]: No worry, look at the PR list, most of them come from maintainers or bots. I have never updated the `pdm` formula myself.

**Comment by VoVAllen** [comment]
Why we didn't use the package `vue3-tabs-component` directly? We can do `pnpm i vue3-tabs-component` to add this package, and use it directly instead of copy the source code

**Comment by frostming** [comment]
> Why we didn't use the package `vue3-tabs-component` directly? We can do `pnpm i vue3-tabs-component` to add this package, and use it directly instead of copy the source code
> 
> 
> 
> Then 
> 
> ```
> 
> import {Tabs, Tab} from 'vue3-tabs-component';
> 
> ```
> 
> and
> 
> ```
> 
>     app
> 
>       .component('CustomTitle', CustomTitle)
> 
>       .component('Author', Author)
> 
>       .component('tabs', Tabs)
> 
>       .component('tab', Tab)
> 
> ```
> 
> Should work 

we used to(see previous commits), but the package isn't well compatible with es modules and can't be built with the new nodejs. However, i am open to revert the change if someone could make it work.

**Comment by VoVAllen** [comment]
@frostming Thanks for your contribution! I tested the core reason is probably the vue3-tab-componenet library is not SSR-compatiable

**Pull request merged**